app/src/pages/playground/__tests__/playgroundUtils.test.ts

```diff
@@ -98,12 +98,12 @@ const defaultTemplate = {
     {
       id: 0,
       role: "system",
-      content: "You are a chatbot",
+      content: "Welcome to Phoenix Playground!\n\nYou are a helpful assistant.",
```

Comment:
Not sure how this helps the user - it's just more text to have to delete

Reply:
Also "Welcome to Phoenix Playground!" shouldn't be a part of the system prompt?

Reply:
It's not necessary, can close.